### DISCUSSION TITLE
Multiplayer game

### DISCUSSION THREAD

**Discussion by jordyBonnet** [pokepetter/ursina#133] [Ideas]
Hello,
Nice job for this open source game engine 👍 .

I tried to find a way to build a multiplayer game based on Ursina, but I found nothing on the API doc or YouTube or anywhere :(

Do you plan to add it at some point in the development ?

Or if it possible with Python builtin "request" module (or flask), Could someone share a quick demo on how to do it ?
A simple one Host + multiple other player able to join with the IP of the host version would be perfect ^^

Regards,

**Comment by ghost** [comment]
Hi jordy, I dont know how to do that but do you know how to make room, like empty cube or something.

**Comment by AshishPandagre** [comment]
Hey @jordyBonnet, 
I don't know how it can be done with request module, but I'm pretty sure it can be done with sockets in python.

**Comment by xzripper** [comment]
<a href="https://github.com/xzripper/ursina_multiplayer">try this</a>
